xdrip-get-entries.sh

```diff
@@ -2043,4 +2043,10 @@ function bt_watchdog()
   fi
 }
 
+function round()
+{
+  echo "$1" | awk '{printf("%d\n",$1 + 0.5)}'
```

Comment:
we always work in mgdl, right? maybe a comment to that effect here would be good since memories fade.
otherwise lgtm!

Reply:
Yes, Logger does everything in mg/dl. Input in mmol are converted to mg/dl and all calcs / output are in mg/dl. NS can change it back to mmol for us depending on user preferences.

I'll add this in a comment in the code as well. Thanks!